File: app/src/main/java/com/malenalbc/superherotest/data/model/Character.java
package com.malenalbc.superherotest.data.model;


import java.util.List;

public class Character extends Result{
    private String name;
    private String description;
    private List<Url> urls;
    private Image thumbnail;

    public String getName () {
        return name;
    }

    public String getDescription () {
        return description;
    }

    public List<Url> getUrls () {
        return urls;
    }

    public Image getThumbnail () {
        return thumbnail;
    }
}

File: data/src/main/java/com/malenalbc/stormcomics/data/model/character/Character.kt
package com.malenalbc.stormcomics.data.model.character

import com.malenalbc.stormcomics.data.model.BaseResult
import java.io.Serializable

data class Character(
    val name: String? = null,
    val description: String? = null,
    val urls: List<Url>? = null,
    val thumbnail: Image? = null
) : BaseResult(), Serializable
